### PR TITLE
chore: provide browser bin via capabilities or env var

### PR DESCRIPTION
**Pull request by sadym-chromium**
Required for running e2e via chromedriver, as chromedriver uses browser binary form capabilities.

### CODE REVIEW

File: tools/path-getter/path-getter.mjs [hunk 1]
```diff
@@ -18,40 +18,54 @@
...
+ * needed.
  * @return {string}
  */
 export function installAndGetChromePath() {
-  let BROWSER_BIN = process.env.BROWSER_BIN;
-  if (!BROWSER_BIN) {
-    BROWSER_BIN = execSync(
-      ['node', join('tools', 'install-browser.mjs')].join(' ')
-    )
+  if (process.env.BROWSER_BIN) return process.env.BROWSER_BIN;
```
Comment (Lightning00Blade) [inline tools/path-getter/path-getter.mjs:32]:
```suggestion
  if (process.env.BROWSER_BIN) {
    return process.env.BROWSER_BIN;
  }
```

File: tools/path-getter/path-getter.mjs [hunk 2]
```diff
@@ -18,40 +18,54 @@
...
  */
 export function installAndGetChromeDriverPath() {
-  let CHROMEDRIVER_BIN = process.env.CHROMEDRIVER_BIN;
-  if (!CHROMEDRIVER_BIN) {
-    CHROMEDRIVER_BIN = execSync(
-      ['node', join('tools', 'install-browser.mjs'), '--chromedriver'].join(' ')
-    )
-      .toString()
-      .trim();
+  if (process.env.CHROMEDRIVER_BIN) return process.env.CHROMEDRIVER_BIN;
```
Comment (Lightning00Blade) [inline tools/path-getter/path-getter.mjs:56]:
```suggestion
  if (process.env.CHROMEDRIVER_BIN) { return process.env.CHROMEDRIVER_BIN;}
```

**Review by Lightning00Blade — APPROVED** [review]
LGTM + 2 comments.

File: tools/path-getter/path-getter.mjs (hunk 1 above)
Comment (sadym-chromium) [reply tools/path-getter/path-getter.mjs:32]: eslint and prettier allow this syntax

Comment (Lightning00Blade) [reply tools/path-getter/path-getter.mjs]: I may be wrong but adding brackets is encouraged by Google style.

**Comment by sadym-chromium** [comment]
https://github.com/GoogleChromeLabs/chromium-bidi/issues/2056

**Pull request merged**